src/ansys/pre_commit_hooks/add_license_headers.py

```diff
@@ -33,12 +33,19 @@
 import os
 import pathlib
 from platform import python_version
+import re
 import shutil
 import sys
 from tempfile import NamedTemporaryFile
 
 import git
-from reuse import _annotate, _util, lint, project
+from reuse import _IGNORE_DIR_PATTERNS, _annotate, _util, lint, project
+
+OWN_IGNORES = [
+    re.compile(r"^\.venv$"),
+]
+_IGNORE_DIR_PATTERNS.extend(OWN_IGNORES)
+
```

Comment:
Agreed with @dipinknair... this is very specific to our naming convention. But it is a start. IMO you should compile the entire .gitignore file.

Reply:
When working with Kerry on the changes in reuse, I saw that they already have something implemented to handle different version control systems. Here is the strategy associated to Git https://github.com/fsfe/reuse-tool/blob/37f342d96b11419516ecc3d7c25fac0fd916c727/src/reuse/vcs.py#L94 and if we are not already leveraging that, then we should probably do it :)

Reply:
Ah that's perfect. Thanks for the heads up @SMoraisAnsys

Reply:
Introduced in #217